.azure/scripts/build-wheels.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -e -x
+
+# Collect the pythons
+pys=(/opt/python/*/bin)
+
+# Filter out Python 3.4
+pys=(${pys[@]//*34*/})
+pys=(${pys[@]//*27*/})
+
+# Compile wheels
+for PYBIN in "${pys[@]}"; do
+    ls -l /io/dist
+    "${PYBIN}/pip" install -r /io/dev-requirements.txt
+    "${PYBIN}/pip" wheel /io/dist/$package_name-*.tar.gz -w wheelhouse/
+done
+
+# Bundle external shared libraries into the wheels
+for whl in wheelhouse/$package_name-*.whl; do
+    auditwheel repair --plat $PLAT "$whl" -w /io/wheelhouse/
+done
+
+# Install packages and test
+for PYBIN in "${pys[@]}"; do
+    "${PYBIN}/python" -m pip install $package_name --no-index -f /io/wheelhouse
+    # Manylinux does not have a JVM so there is no way to test the wheel in the docker
```

Comment:
Have we had the discussion already, if we should derive from Manylinux for this very purpose?

Reply:
I think you may have mentioned it.  Though I am not sure I can pull it off.  My reading of manylinux is that it is some ancient version of redhat.   It would be hard to get a JVM on that version.  A better approach would be to add a second docker task or use azure itself to test the install.  Unfortunately I ran out of gas after 10 hours of working on the release and just decided to punt.   The full pipeline of linux is pretty slow so to develop this we would need to have test pipeline that pulls the artifacts from the earlier build, works out all the patterns then adds it to the release.

Reply:
Many thanks for this massive effort!
It used to be derived from redhat, now it is debian stretch (oldstable). So in principle we should be able to get a jdk. But I guess you're right, we should test it outside of the manylinux container. We can postpone it for later efforts on the __great__ release pipeline